analyzer/dhcp/CMakeLists.txt

```diff
@@ -1,7 +1,13 @@
 # Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
 
 spicy_add_analyzer(
-    NAME DHCP
-    PACKAGE_NAME ${PACKAGE_NAME}
-    SOURCES dhcp.spicy dhcp.evt zeek_dhcp.spicy
-    SCRIPTS __load__.zeek)
+    NAME
+    DHCP
+    PACKAGE_NAME
+    ${PACKAGE_NAME}
+    SOURCES
+    dhcp.spicy
+    dhcp.evt
+    zeek_dhcp.spicy
+    SCRIPTS
+    __load__.zeek)
```

Comment:
I don't like this formatting at all ... Any chance to configure this differently?